src/System.Runtime.CompilerServices.VisualC/ref/System.Runtime.CompilerServices.VisualC.csproj

```diff
@@ -3,10 +3,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Runtime.CompilerServices.VisualC.cs" />
+     <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj"/>
```

Comment:
NIT: looks like indenting is wrong in this line.

Can you add a ToDo comment to comment to remove this P2P reference once we update the System.Runtime package we use for this library? Also if you can add an issue to fix that and put the issue in the comment that would be great